packages/ckeditor5-table/src/tableproperties/ui/tablepropertiesview.js

```diff
@@ -360,8 +360,10 @@ export default class TablePropertiesView extends View {
 		[
 			this.borderStyleDropdown,
 			this.borderColorInput,
+			this.borderColorInput.fieldView._dropdownView.buttonView,
```

Comment:
I think this is the right moment do make `_dropdownView` public in the component. `ColorInputView` is private anyway (for now) so this will not hurt anyone.